github/PullRequest.py

```diff
@@ -409,25 +409,67 @@ def create_comment(self, body, commit_id, path, position):
         """
         return self.create_review_comment(body, commit_id, path, position)
 
-    def create_review_comment(self, body, commit_id, path, position):
+    def create_review_comment(
+        # line replaces deprecated position argument, so we put it between path and side
+        self,
+        body,
+        commit,
+        path,
+        line=None,
```

Comment:
let's move this into place, now that this is multiline:
```suggestion
        self,
        body,
        commit,
        path,
        # line replaces deprecated position argument, so we put it between path and side
        line=None,
```